src/main/scala/App/Main.scala

```diff
@@ -40,6 +41,11 @@ object Main extends App {
 
   println("-------- All Users Deleted ---------------")
   println(userRepo.getAll)
+
```

Comment:
Why have you added these extra lines?

Reply:
These line were already in the code, but when I was reviewing my code I think then these lines may be touched or formated

Reply:
Here, I was just checking something for the UserDB mocking

Reply:
Kindly remove the extra lines added.